skglm/solvers/cd_solver.py

```diff
@@ -4,7 +4,7 @@
 from sklearn.utils import check_array
 
 
-def cd_solver_path(X, y, datafit, penalty, alphas=None,
+def cd_solver_path(X, y, datafit, penalty, alphas=None, fit_intercept=True,
```

Comment:
to discuss: this is the sklearn default setting, but it requires all datafits to implement fit_intercept. And it's usually confusing for the user. I'd set it to False even if it disagrees with sklearn

Reply:
agreed!